snc.sh

```diff
@@ -191,12 +191,12 @@ ${SSH} core@api.${SNC_PRODUCT_NAME}.${BASE_DOMAIN} sudo hostnamectl set-hostname
 
 create_json_description ${BUNDLE_TYPE}
 
-# Create persistent volumes
-create_pvs ${BUNDLE_TYPE}
-
 # Install LVMS operator
 install_lvms_operator ${BUNDLE_TYPE}
 
+# Create persistent volumes
+create_registry_pvc
```

Comment:
It's not creating persistent volumes, it's creating the registry persistent volume claims (I'm mostly pointing out that the comments are paraphrasing the function names, and here the method was renamed, but the comment not updated. Imo these comments have very little added value since the function names are very explicit).